exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/H2.kt

```diff
@@ -153,15 +153,15 @@ open class H2Dialect : VendorDialect(dialectName, H2DataTypeProvider, H2Function
 
     override fun toString(): String = "H2Dialect[$dialectName, $h2Mode]"
 
-    internal enum class H2MajorVersion {
+    enum class H2MajorVersion {
         One, Two
     }
 
     internal val version by lazy {
         exactH2Version(TransactionManager.current())
     }
 
-    internal val majorVersion: H2MajorVersion by lazy {
+    val majorVersion: H2MajorVersion by lazy {
```

Comment:
It's not just about including it in tests. The function `checkUnmappedSequences` in `MigrationUtils` would return unwanted DROP statements if we don't do this check, and the check needs these properties to be public to be done. Until we completely drop support for H2_V1, I wouldn't want users to have those unwanted DROP statements since it could really mess up their database. Since we're phasing out support for H2_V1 anyway, it won't matter that these properties are public because they're going to be gone soon anyway.